src/include/index/vamana_index.h

```diff
@@ -647,13 +736,13 @@ class vamana_index {
       const Q& query_set,
       size_t k,
       std::optional<size_t> opt_L = std::nullopt) {
-    scoped_timer __{tdb_func__ + std::string{" (outer)"}};
+    scoped_timer __{tdb_func__ + std::string{" (outer)"}, true};
 
     size_t L = opt_L ? *opt_L : L_build_;
     // L = std::min<size_t>(L, L_build_);
 
-    auto top_k = ColMajorMatrix<id_type>(k, ::num_vectors(query_set));
-    auto top_k_scores = ColMajorMatrix<score_type>(k, ::num_vectors(query_set));
+    auto top_k = ColMajorMatrix<size_t>(k, ::num_vectors(query_set));
+    auto top_k_scores = ColMajorMatrix<float>(k, ::num_vectors(query_set));
```

Comment:
Why change this back? I had to make this change else the new Python unit test would fail, was that incorrect? https://github.com/TileDB-Inc/TileDB-Vector-Search/pull/229/files#diff-1f60371badd7cdf91f0e4ca43c1e1bd20ff2856550c53f87924ee8c9bab46a22R655

Reply:
This was an error in a merge step and should not have been changed back.  Good catch.